### DISCUSSION TITLE
Upcoming Changes

### DISCUSSION THREAD

**Discussion by lrljoe** [LowerRockLabs/LaravelLivewireTablesAdvancedFilters#73] [Announcements]
Several upcoming improvements as a result of changes to the Core Package from Rappasoft

### Improvements
- Simplification of AlpineJS components/calls
- General clean-up of blades to improve performance

### New Features
- Introduction of a Component Filter to allow for further extensibility.

### Documentation
- Further documentation on using the package

**Comment by BenOussama180** [comment]
cant wait
